zubhub_frontend/zubhub/src/views/create_activity/CreateEditActivity.jsx

```diff
@@ -42,7 +43,8 @@ const steps = ['Activity Basics', 'Activity Details'];
 
 let firstRender = true;
 
-export default function CreateActivity(props) {
+function CreateEditActivity(props) {
+  const editting = props?.editting;
```

Comment:
added an `editting` property to handle data editting and change in words titles, button labels.